protocol/synthetix/contracts/modules/core/RewardsManagerModule.sol

```diff
@@ -242,6 +242,10 @@ contract RewardsManagerModule is IRewardsManagerModule {
         }
         pool.vaults[collateralType].rewards[rewardId].distributor = IRewardDistributor(address(0));
 
+        RewardDistribution.Data storage reward = pool.vaults[collateralType].rewards[rewardId];
```

Comment:
I think that this can be put above line 243, and we can use the storage pointer in that line too. That way we can save a storage read, which are really gas expensive.